lib/at_cmd_parser/include/at_utils.h

```diff
@@ -0,0 +1,45 @@
+/**@file at_utils.h
+ *
+ * @brief AT parser utility functions to deal with strings.
+ *
+ */
+
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef AT_UTILS_H_
+#define AT_UTILS_H_
+
+#include <stdint.h>
```

Comment:
`u32_t` is not defined in `stdint.h`. Include `zephyr/types.h` instead.

Reply:
Fixed